lib/hippie/client.js

```diff
@@ -13,6 +13,7 @@ var http = require('http');
 var serializers = require('./serializers');
 var parsers = require('./parsers');
 var expect = require('./expect');
+var Promise = require('es6-promise').Promise;
```

Comment:
@CacheControl 3rd is what I would expect as a user of the library

@logicalparadox your thoughts?